topics/single-cell/tutorials/pseudobulk-analysis/tutorial.md

```diff
@@ -66,23 +69,29 @@ In this tutorial, we will guide you through a pseudobulk analysis workflow using
 >
 {: .agenda}
 
-# Let's Get the Data!
+# Get the Data!
 
 ## Overview of the Data
 
-Our data was extracted from the publication titled _"Elevated Calprotectin and Abnormal Myeloid Cell Subsets Discriminate Severe from Mild COVID-19"_ ({% cite Silvin2020 %}). This dataset was chosen because it was utilized by the developers of the `decoupler` tool for pseudobulk aggregate analysis ({% cite decoupler-pseudobulk %}).
+The dataset used in this tutorial is an `AnnData` object containing a subset of single-cell RNA-seq data, specifically including non-classical monocytes (NCMs) and plasmacytoid dendritic cells (pDCs) from bone marrow and peripheral blood samples. This subset was selected to illustrate differential gene expression between two immune cell types across two biologically distinct tissues.
+
+In this tutorial, we will focus on these two cell populations and explore how their expression profiles differ between bone marrow and blood using pseudobulk analysis.
+
+Pseudobulk analysis is an advanced approach in single-cell data analysis. It involves aggregating single-cell expression data by group (e.g., by cell type and condition) to perform bulk-style analyses while preserving single-cell resolution through metadata.
+
+For this tutorial, we assume familiarity with standard single-cell data formats, such as `AnnData` or `Seurat` objects, and prior experience with single-cell workflows, particularly clustering and cell type annotation. We will use the `decoupler` tool to perform the pseudobulk aggregation and downstream differential analysis {% cite decoupler-pseudobulk %}.
```

Comment:
Describe some basic numbers on total number cells or number of cells per condition etc. Otherwise, it is also good to show those numbers after data upload, at the very beginning of the analysis.